pkg/local_instance.go

```diff
@@ -134,7 +138,7 @@ func (li LocalInstance) overrideDirsChecksum() (string, error) {
 }
 
 func (li LocalInstance) LockDir() string {
-	return pathx.Canonical(fmt.Sprintf("%s/%s/lock", li.WorkDir(), common.VarDirName))
+	return pathx.Canonical(fmt.Sprintf("%s/%s/state", li.WorkDir(), common.VarDirName))
```

Comment:
This makes existing instance not managed / breaking Change

Reply:
reverted... at least for now